tests/lib/fakestore/store/store.go

```diff
@@ -218,7 +218,7 @@ func snapEssentialInfo(w http.ResponseWriter, fn, snapID string, bs asserts.Back
 	}
 
 	return &essentialInfo{
-		Name:        info.Name(),
+		Name:        info.StoreName(),
```

Comment:
I think InstanceName would also have been fine

Reply:
Ginve that it's sort of mocking the store side of things, wouldn't it be better/clearer for this to remain as `StoreName()`?

Reply:
it's probably subjective, but given that I still hope we might s/InstanceName/Name/, StoreName becomes, I know there's likely a instance key here in play that I don't want